.gitignore

```diff
@@ -40,3 +40,20 @@ objs/
 # Other
 *~
 GeneratedCode
```

Comment:
what is this folder?
Note: if we want to ignore a folder we should add a / at the end eg `GeneratedCode/`

Reply:
i think it is an old folder from opendds